radio/src/gui/480x272/lcd.cpp

```diff
@@ -280,7 +280,7 @@ void drawTimerMode(coord_t x, coord_t y, int32_t mode, LcdFlags att)
 {
   if (mode >= 0) {
     if (mode < TMRMODE_COUNT)
-      return lcdDrawTextAtIndex(x, y, STR_VTMRMODES, mode, att);
+      lcdDrawTextAtIndex(x, y, STR_VTMRMODES, mode, att);
```

Comment:
Bertrand, so this is a good place to ask about style....

Are you OK with this?

``` c
if (simple)
  foo();

// or

if (simple)
  foo();
else
  bar():
```

Because I personally dislike the one-line versions `if (simple) foo();` and putting in the curly braces for very simple cases seems a bit verbose at times (especially in the first example).  I agree that if braces are used in one part of the condition (eg. the `if`) then they should be used for the other part(s) (the `else`), as you say.